modules/google-speech/src/backend/client.ts

```diff
@@ -81,7 +81,12 @@ export class GoogleSpeechClient {
   }
 
   public async speechToText(audioFileUrl: string, language: string, timeout?: string): Promise<string | undefined> {
-    debugSpeechToText('Received audio to recognize:', audioFileUrl)
+    debugSpeechToText('Received audio file to recognize:', audioFileUrl)
+
+    // Media Service URLs
+    if (!audioFileUrl.startsWith('http')) {
```

Comment:
possible that this won't work the day we start having an s3 backend to media service .... media on s3 start by http as well

Reply:
Good call, I'll simply use the isBpUrl function available through `common/url`. Simply forgot it existed!